<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>330</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 194, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>21</y>
      <width>591</width>
      <height>245</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelPATH">
         <property name="text">
          <string>PATH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_browser">
         <property name="text">
          <string>Browser</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QQuickWidget" name="quickWidget">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="resizeMode">
            <enum>QQuickWidget::SizeRootObjectToView</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_duration">
             <property name="text">
              <string>_:_</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider_duration">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_total">
             <property name="text">
              <string>_:_</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_stop">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color: rgb(180, 180, 180); }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/Icon/stop.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_previous">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color: rgb(180, 180, 180); }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/Icon/previous.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_play">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color: rgb(180, 180, 180); }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/Icon/play.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_next">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color: rgb(180, 180, 180); }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/Icon/next.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>208</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSlow"/>
    <addaction name="actionFast"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Icon/open.png</normaloff>:/Icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Icon/exit.png</normaloff>:/Icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSlow">
   <property name="text">
    <string>Slow</string>
   </property>
  </action>
  <action name="actionFast">
   <property name="text">
    <string>Fast</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_browser</tabstop>
  <tabstop>horizontalSlider_duration</tabstop>
  <tabstop>quickWidget</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
